src/main/java/frc/trigon/robot/utilities/Conversions.java

```diff
@@ -0,0 +1,64 @@
+package frc.trigon.robot.utilities;
+
+public class Conversions {
+    private static final double MAG_TICKS = 4096;
+    private static final double FALCON_TICKS = 2048;
+
+    public static double magTicksToDegrees(double magTicks) {
+        return magTicksToRevolutions(magTicks) * 360;
+    }
+
+    public static double degreesToMagTicks(double degrees) {
+        return degreesToRevolutions(degrees) * MAG_TICKS;
+    }
+
+    public static double degreesToRevolutions(double degrees) {
+        return degrees / 360;
+    }
+
+    public static double revolutionsToDegrees(double rotations) {
+        return rotations * 360;
+    }
+
+    public static double revolutionsToMag(double rotations) {
+        return rotations * MAG_TICKS;
+    }
+
+    public static double magTicksToRevolutions(double MagTicks) {
+        return MagTicks / MAG_TICKS;
+    }
+
+    public static double mpmsToMps(double MagTicks) {
```

Comment:
mpms?

Reply:
meters per mele second

Reply:
Why would you want that?
Just have a function to convert ms to s